mypyc/build.py

```diff
@@ -534,6 +534,8 @@ def mypycify(
             "-Wno-unused-command-line-argument",
             "-Wno-unknown-warning-option",
             "-Wno-unused-but-set-variable",
+            "-Wno-cpp",
```

Comment:
Can you add a short comment here explaining what this does (and perhaps include a link to the GitHub issue).